lib/template.js

```diff
@@ -1,24 +1,34 @@
 var mustache = require('mustache'),
     _ = require('underscore'),
     moment = require('moment'),
-    config = require('../config');
+    config = require('../config'),
+    toBikramSambatLetters = require('bikram-sambat').str_toBik_text;
 
-var format = function(date, key) {
+var configuredFormat = function(date, key) {
+  return format(date, config.get(key));
+};
+
+var format = function(date, formatString) {
   if (!isNaN(date)) {
     date = parseInt(date, 10);
   }
-  return moment(date).format(config.get(key));
+  return moment(date).format(formatString);
 };
 
 var formatters = {
+  bikram_sambat_date: function() {
```

Comment:
I wonder if this can be made more concise.  Good to have `_date` in there for people who don't know what `bikram_sambat` means; but would e.g. `bikram_date` make sense to Nepali tech leads/project managers?